src/client/pages/Landing/components/Card.tsx

```diff
@@ -76,7 +76,7 @@ const CardHeader = styled.div`
     display: none;
   }
 
-  @media (min-height: 620px) {
+  @media (min-height: 600px) {
```

Comment:
Wasn't all `@media (min-height:` set to `620px` before? :) Or maybe this change makes sense, just wanted to check that you don't mix it up with the `min-width` media query that's set to `600px`

Reply:
Good call! Will change back